proto/eth/v2/custom.go

```diff
@@ -65,3 +66,32 @@ func (x *LightClientHeaderContainer) GetBeacon() (*v1.BeaconBlockHeader, error)
 		return nil, fmt.Errorf("unknown header type: %T", input)
 	}
 }
+
+func (x *LightClientHeaderContainer) GetExecutionHeaderCapella() (*enginev1.ExecutionPayloadHeaderCapella, error) {
+	switch input := x.Header.(type) {
+	case *LightClientHeaderContainer_HeaderCapella:
+		return input.HeaderCapella.Execution, nil
+	default:
+		return nil, fmt.Errorf("header type not capella: %T", input)
```

Comment:
```suggestion
		return nil, fmt.Errorf("header type %T not Capella", input)
```
Semicolons separate errors in Go so it's a best practice not to used them within error messages